cartographer_ros/configuration_files/revo_lds.lua

```diff
@@ -24,6 +24,9 @@ options = {
   odom_frame = "odom",
   provide_odom_frame = true,
   use_odometry = false,
+  fixed_frame_pose_frame = "fixed_frame_pose",
+  provide_fixed_frame_pose_frame = true,
+  use_fixed_frame_pose = false,
```

Comment:
If there are new options, defaults also need to be set in https://github.com/googlecartographer/cartographer_toyota_hsr/tree/master/cartographer_toyota_hsr/    https://github.com/googlecartographer/cartographer_turtlebot   https://github.com/googlecartographer/cartographer_fetch/tree/master/cartographer_fetch/configuration_files in follow-up PRs.